ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -72,27 +72,6 @@ import           Ouroboros.Network.Point (WithOrigin (..), block, origin,
                      withOriginToMaybe)
 import qualified Ouroboros.Network.Point as Point (Block (..))
 
--- | The 0-based index for the Ourboros time slot.
-newtype SlotNo = SlotNo { unSlotNo :: Word64 }
-  deriving stock   (Show, Eq, Ord, Generic)
-  deriving newtype (Enum, Bounded, Num, Serialise, NoUnexpectedThunks)
-
-instance ToCBOR SlotNo where
-  toCBOR = encode
-
--- | The 0-based index of the block in the blockchain.
--- BlockNo is <= SlotNo and is only equal at slot N if there is a block
--- for every slot where N <= SlotNo.
-newtype BlockNo = BlockNo { unBlockNo :: Word64 }
-  deriving stock   (Show, Eq, Ord, Generic)
-  deriving newtype (Enum, Bounded, Num, Serialise, NoUnexpectedThunks)
-
-genesisSlotNo :: SlotNo
-genesisSlotNo = SlotNo 0
```

Comment:
See #1296. That's now something that will have to be addressed in `cardano-base/prelude`.